src/edu/cmu/tartan/Game.java

```diff
@@ -150,6 +146,14 @@ public void start() throws NullPointerException {
                 input = this.scannerInInterface.nextLine();
 
                 if (input.compareTo("quit") == 0) {
+					saveScheduler.cancel();
+
+                	printOutInterface.console("Do you want to save the game? (Yes/No)");
+                	String response = scannerInInterface.nextLine();
+
+                	if(response.equalsIgnoreCase("yes") || response.equalsIgnoreCase("y")){
```

Comment:
It is likely that you need to add processing for yes or not.
Currently, it is likely to be treated as no | n even if abcd is entered instead of no | n.
And it seems to be correct to run the while statement until the normal input value is input.